src/error/mod.rs

```diff
@@ -160,6 +167,14 @@ pub enum EvalexprError {
         divisor: Value,
     },
 
+    /// This regular expression could not be parsed
```

Comment:
Comments of the error types are usually worded for the indefinite case. This is how I do it in this crate, not sure if it's the best way. But we should keep it consistently.

```suggestion
    /// A regular expression could not be parsed
```

Reply:
Fixed.